admin/services/data-access/check.data.service.js

```diff
@@ -46,6 +87,30 @@ checkDataService.findFullyPopulated = async function (criteria) {
     .exec()
 }
 
+/**
+ * Find Checks and associated data by checkCode
+ * @param checkCodes - array of UUID
+ * @return {Promise.<void>} - lean Check objects, fully populated
+ * This includes the pupil (includes the school), checkWindow, and checkForm.  This is fairly efficient
+ * involving 1 extra query per document set per sub-document.  The whole lot is done in 5 queries total.
+ */
+checkDataService.sqlFindFullyPopulated = async function (checkCodes) {
+  let sql = 'SELECT * FROM mtc_admin.[check] chk INNER JOIN mtc_admin.pupil pup ON pup.id = chk.pupil_id INNER JOIN mtc_admin.school sch ON sch.id = pup.school_id INNER JOIN mtc_admin.checkWindow wdw ON wdw.id = chk.checkWindow_id INNER JOIN mtc_admin.checkForm frm ON frm.id = chk.checkForm_id'
```

Comment:
This is a bit long - would you consider using backticks  and breaking it up to be more legible?

Reply:
👍